vortex-api/src/main/java/mil/army/usace/hec/vortex/util/VortexTimeUtils.java

```diff
@@ -0,0 +1,25 @@
+package mil.army.usace.hec.vortex.util;
+
+import java.time.Duration;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+
+public class VortexTimeUtils {
+    private static final ZoneId BASE_ZONE_ID = ZoneId.of("UTC");
+    public static final ZonedDateTime BASE_TIME = ZonedDateTime.of(1900, 1, 1, 0, 0, 0, 0, BASE_ZONE_ID);
+
+    public static long getNumDurationsFromBaseTime(ZonedDateTime dateTime, ChronoUnit minimumUnit) {
```

Comment:
Would this be the number of intervals? Not sure what is meant by `Durations` in the method name?

Reply:
Renamed to `getNumIntervalsFromBaseTime` and `minimumUnit` to `intervalUnit`

Reply:
I might use "Origin time" rather than "Base time" to be more explicit.

Reply:
Renamed